index.html

```diff
@@ -0,0 +1,127 @@
+                <h2 class="headline-date">2022.12.15(THU) ~ 16(FRI)</h2>
+                <p class="headline-last-p">@National Museum of Afghanistan, Art Center and More</p>
+            </div>
+        </section>
+        <section class="program-section">
+            <h3>Main Program</h3>
+            <div class="prog-main-cont">
+                <div class="prog-card">
+                    <img class="prog-img" src="./img/lecture.png" alt="">
+                    <h4 class="prog-title">Lecture</h4>
+                    <p class="prog-content">
+                        It is a long established fact that a reader will be distracted 
+                        by the readable content of a page when looking at its layout.
+                    </p>
+                </div>
+                <div class="prog-card">
+                    <img class="prog-img" src="./img/exhibition.png" alt="">
+                    <h4 class="prog-title">CC Exhibition</h4>
+                    <p class="prog-content">
+                        It is a long established fact that a reader will be distracted 
+                        by the readable content of a page when looking at its layout.
+                    </p>
+                </div>
+                <div class="prog-card">
+                    <img class="prog-img" src="./img/camunication.png" alt="">
+                    <h4 class="prog-title">Forum</h4>
+                    <p class="prog-content">
+                        It is a long established fact that a reader will be distracted 
+                        by the readable content of a page when looking at its layout.
+                    </p>
+                </div>
+                <div class="prog-card">
+                    <img class="prog-img" src="./img/workshop.png" alt="">
+                    <h4 class="prog-title">Workshop</h4>
+                    <p class="prog-content">
+                        It is a long established fact that a reader will be distracted 
+                        by the readable content of a page when looking at its layout.
+                    </p>
+                </div>
+                <div class="prog-card">
+                    <img class="prog-img" src="./img/ignite.png" alt="">
+                    <h4 class="prog-title">CC Ignite</h4>
+                    <p class="prog-content">
+                        It is a long established fact that a reader will be distracted 
+                        by the readable content of a page when looking at its layout.
+                    </p>
+                </div>
+            </div>
+            <a class="prog-hole" href="#">See the Hole Program</a>
+            <button class="prog-btn">Joid CC Global Summit 2015</button>
+        </section>
+        <section class="speaker-section">
+            <h3 class="speaker-title">Featured Speakers</h3>
+            <div class="speaker-main-cont">
+                <div class="speaker-cards">
+                </div>
```

Comment:
- [ ] The bottom section should include a section header, a divider, and six individual articles, kindly consider adding the divider below the section Header
- [ ] The bottom section articles should be organized in two columns of three articles each, kindly make sure to follow this rule so the design looks identical to the provided one

|Your Page|Provided design|
|-|-|
|![Screenshot_20221203_093044](https://user-images.githubusercontent.com/25227022/205432188-b80521c1-5a89-4d40-a551-9a79e20e5b55.png)|![Screenshot_20221203_093130](https://user-images.githubusercontent.com/25227022/205432195-f0609a58-7ce7-47ec-8e74-db783d9fee0a.png)|